This notebook explores the PPP data from https://data.sba.gov/dataset/ppp-foia/resource/6dd520ff-c211-404f-b7eb-29784cd4dbda?inner_span=True.

The goal is to see how PPP disbursements compared to the number of US religious congs from the US religious census: https://www.usreligioncensus.org/, with a special focus on whether black church congregrations didn't benefit from the PPP program as much as other groups.

One major difficulty in this analysis is that the Census and PPP data use different names for the groups. The PPE uses individual congregrations by their legal (TBA) names, but the Census uses scholarly / official names, which can be different. For example, the Census identifies five groups as "Anglican":
    Anglican Catholic Church
    Anglican Church in America
    Anglican Church in North America
    Anglican Province of America
    Anglican Province of Christ the King
 
The "Anglican Church in NA" has the most congregrations (873), while the other four have a total of less than 200. None of these report the number of adherents. But most "Anglican" churches names don't use these names. To which group does "St. Paul's Anglican Church" belong? The PPP data doesn't allow us to determine this.

Similarly, the Census has data about different Jewish groups ("Reform Judaism", "Conservative Judaism"), but many synagogues don't use these terms in their names: is "Temple Israel" a reform, conservative, or some other branch of Judaism?

In this analysis, I have endeavored to address these issues and while there are doubtless errors, I think the overall conclusions are correct.


In [1]:
import pandas as pd
pd.options.display.max_colwidth = 400
pd.options.display.max_columns = 90

In [2]:
import glob
workDir = '/home/alp/Google Drive/Python/analysis/PPEdata/'

file_names = sorted(glob.glob(workDir+'public_up_to*.csv'))
firstFile = 'public_150k_plus_220930.csv'

colList = ['BorrowerName','BorrowerState','BorrowerZip','CurrentApprovalAmount','JobsReported','NAICSCode','Race','Ethnicity','BusinessType','ForgivenessAmount']

df = pd.read_csv(workDir+firstFile, usecols = colList)

for file in file_names:
    oldDF = df
    print(file)
    if firstFile not in file:
        newDF = pd.read_csv(file, usecols = colList)
        df = pd.concat([oldDF,newDF])

df.info()        

/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_10_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_11_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_12_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_1_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_2_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_3_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_4_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_5_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_6_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_7_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_8_220930.csv
/home/alp/Google Drive/Python/analysis/PPEdata/public_up_to_150k_9_220930.csv
<class 'pandas.core.frame.DataFrame'>
Int64Index: 11468438 en

In [3]:
print(f'number of records in the dataframe is {len(df):,}')

number of records in the dataframe is 11,468,438


How to separate out the religious organizations? 

The PPP incldues a field for NAICS codes. 813110 is the NAICS code for religious organizations -- https://www.census.gov/naics/?input=813110&year=2022&details=813110:
```
    "This industry comprises (1) establishments primarily engaged in operating religious organizations, such as churches, religious temples, mosques, and monasteries, and/or (2) establishments primarily engaged in administering an organized religion or promoting religious activities."
```

In [4]:
print('pct of records disbursed to religious organizations:')
len(df[df['NAICSCode'] == 813110])/len(df)*100

pct of records disbursed to religious organizations:


1.0695964001374905

We will create a new DF just for religious organizations and clean it a bit.

In [5]:
rel_df  = df[df['NAICSCode'] == 813110]
len(rel_df)

122666

In [6]:
rel_df['BorrowerName'].isna().sum()

2

In [7]:
rel_df = rel_df.dropna(subset = ['BorrowerName'])

The PPP records a wide variety of different types of business, even among the religious organizations. It's hard to know if some of these should be excluded from the analysis (especially since there have been news reports of folks scamming the PPP system __[NBC](https://www.nbcnews.com/politics/justice-department/biggest-fraud-generation-looting-covid-relief-program-known-ppp-n1279664)__ and __[NPR](https://www.npr.org/2022/10/12/1128207464/ppp-loans-loan-forgiveness-small-business)__. But since I have no way of determining if any of the loans reported in the PPP database are fraudulent or if some categories (such as "Sole Proprietorship") might have higher rates of fraud than others (such as "Corporation"), I will retain all of them for this analysis.

In [8]:
rel_df['BusinessType'].value_counts()

Non-Profit Organization                95301
Corporation                            10458
Professional Association                4276
501(c)3 – Non Profit                    3833
Sole Proprietorship                     3177
Limited  Liability Company(LLC)         1520
Self-Employed Individuals               1138
Independent Contractors                 1081
Subchapter S Corporation                 694
Non-Profit Childcare Center              314
Partnership                              213
Cooperative                              127
501(c)6 – Non Profit Membership          125
Trust                                    105
Single Member LLC                         32
Limited Liability Partnership             27
Employee Stock Ownership Plan(ESOP)       16
Joint Venture                              9
501(c)19 – Non Profit Veterans             8
501(c) – Non Profit except 3,4,6,          3
Qualified Joint-Venture (spouses)          3
Tenant in Common                           2
Rollover a

Now, let's create a subset df that extracts just the religious organizations that are among the family of churchs that are traditional black churches. 

According to Frontline(https://www.pbs.org/wgbh/americanexperience/features/godinamerica-black-church/):
```
Today "the black church" is widely understood to include the following seven major black Protestant denominations: 
    the National Baptist Convention, 
    the National Baptist Convention of America, 
    the Progressive National Convention, 
    the African Methodist Episcopal Church, 
    the African Methodist Episcopal Zion Church, 
    the Christian Methodist Episcopal Church and
    the Church of God in Christ.
```
So let's look for those seven denoms.

In [9]:
bc_fam = rel_df[rel_df['BorrowerName'].str.contains('NATIONAL BAPTIST | PROGRESSIVE NATIONAL | AFRICAN METHODIST | METHODIST EPISCOPAL | CHURCH OF GOD IN CHRIST')
                        ]
len(bc_fam)

552

In [10]:
# percent of religion loans represted by BC fam churches:
len(bc_fam)/len(rel_df)*100

0.4500097828213657

In the PPE data, 122,664 loans were provided to religious organizations.

Of this, 552 were provided to congs that may be one of the seven historic BCs. 
This is 0.45% of the total. 

But in the Census, the 7 BC families comprise about 3.77% of all congregrations in the USA.

There maybe innoculous explainations for this disparity (perhaps the BCs have smaller staffs or unpaid ministers) but at least part of the explanation must be the difficulty of applying for PPO funds. To do so required some familarity with the banking industry, legal knowledge helped, etc.

As a check on this, let's examine the data for the Episcopal Church, Hindu, and Jewish groups. These groups have higher-than-average education and income levels; see the __[PEW religious landscape survey](https://www.pewresearch.org/religion/religious-landscape-study/)__ data for information about this. What percent of the national group do they represent? Did they get this amount from the PPO pool or a higher number?

We will start by creating dataframes for the three other groups and then examining how many loans they received.

In [11]:
j_fam = rel_df[(rel_df['BorrowerName'].str.contains('SYNAGOGUE | JEWISH | ISRAEL')) 
                        ]

In [14]:
episcop = rel_df[rel_df['BorrowerName'].str.contains('EPISCOPAL | ANGLICAN' )] 
# but this will pick up the AMEZ church, so we need to delete those rows

episcop = episcop.drop( episcop[episcop['BorrowerName'].str.contains('AFRICAN METHODIST')].index)

In [15]:
hindu = rel_df[rel_df['BorrowerName'].str.contains('HINDU | YOGA')]

In [16]:
print(f'Number of PPE loans disbursed to Jewish congs: {len(j_fam)}')
print(f'Pct of religion loans made to J congs: {len(j_fam)/len(rel_df)*100:.2f}')
print(f'Number of PPE loans disbursed to Episcopal congs: {len(episcop)}')
print(f'Pct of religion loans made to E congs: {len(episcop)/len(rel_df)*100:.2f}')
print(f'Number of PPE loans disbursed to Hindu congs: {len(hindu)}')
print(f'Pct of religion loans made to H congs: {len(hindu)/len(rel_df)*100:.2f}')

Number of PPE loans disbursed to Jewish congs: 1034
Pct of religion loans made to J congs: 0.84
Number of PPE loans disbursed to Episcopal congs: 3034
Pct of religion loans made to E congs: 2.47
Number of PPE loans disbursed to Hindu congs: 182
Pct of religion loans made to H congs: 0.15


Summary:

| Relig group/fam | N PPP loans | Pct PPP loans | Pct All US cong | N loans if proportional |
| --- | --: | --: | --: | --: |
| Black Church | 552 | 0.45 | 3.77 | 4624 |
| Jewish | 1034 | 0.84 | 0.66 | 809 |
| Episcopal | 3034 | 2.47 | 2.03 | 3594 |
| Hindu | 182 | 0.15 | 0.51 | 625 |

This suggests that
- Episcopal groups received *fewer* PPP loans than would be proportional to their number in the Census (roughly -15%).
- Jewish groups recevied *more* loans than would be proportional (+28%).
- Hindu groups received *many fewer* loans than would be proportional (-71%).
- Black churches reeived *many fewer* loans that would be proportional (-88%). 

But perhaps there is a difference in the
1. the size of the congregrations?
1. the number of jobs at risk?
1. the amount of the loans?

Data for the first can be found in the Census:
| Relig group/fam | N congs | N Adherents | Adh per Cong |
| --- | --: | --: | --: |
| Black Church | 13,457 | 4,602,235 | 342 |
| Jewish | 2,353 | 2,068,827 | 879 
| Episcopal | 7,226 | 1,576,611 | 218 |
| Hindu | 1,811 | 1,276,444 | 704 |

This shows that the average size of Episcopal congregrations is the smallest of the four groups, followed by Black churchs. Hindu and Jewish congregrations are considerably larger.

Let's look at the latter two questions:

In [17]:
totBC, totJ, totE, totH = bc_fam['JobsReported'].sum(), j_fam['JobsReported'].sum(), episcop['JobsReported'].sum(),hindu['JobsReported'].sum()
totBCfg, totJfg, totEfg, totHfg = bc_fam['ForgivenessAmount'].sum(), j_fam['ForgivenessAmount'].sum(), episcop['ForgivenessAmount'].sum(),hindu['ForgivenessAmount'].sum()
print(f'{"Cong Family":>15}{"Total N Loans":>15}{"Total Jobs":>15}{"Jobs per loan":>15}{"Total $ Forgiven":>20}{"Forgive per Cong":>20}{"Forgive per Job":>20}{"Forgive per loan":>20}')
print(f'{"Black church":>15}{len(bc_fam):>15n}{totBC:>15n}{(totBC/len(bc_fam)*100):>15,.0f}{totBCfg:>20,.2f}{(totBCfg/len(bc_fam)):>20,.2f}{(totBCfg/totBC):>20,.2f}{(totBCfg/len(bc_fam)):>20,.2f}')
print(f'{"Jewish congs":>15}{len(j_fam):>15n}{totJ:>15n}{(totJ/len(j_fam)*100):>15,.0f}{totJfg:>20,.2f}{(totJfg/len(j_fam)):>20,.2f}{(totJfg/totJ):>20,.2f}{(totJfg/len(j_fam)):>20,.2f}  ')
print(f'{"Episcopal":>15}{len(episcop):>15n}{totE:>15n}{(totE/len(episcop)*100):>15,.0f}{totEfg:>20,.2f}{(totJfg/len(episcop)):>20,.2f}{(totEfg/totE):>20,.2f}{(totJfg/len(episcop)):>20,.2f}  ')
print(f'{"Hindu congs":>15}{len(hindu):>15n}{totH:>15n}{(totH/len(hindu)*100):>15,.0f}{totHfg:>20,.2f}{(totHfg/len(hindu)):>20,.2f}{(totHfg/totH):>20,.2f}{(totJfg/len(hindu)):>20,.2f}  ')


    Cong Family  Total N Loans     Total Jobs  Jobs per loan    Total $ Forgiven    Forgive per Cong     Forgive per Job    Forgive per loan
   Black church            552           3664            664       16,732,818.05           30,313.08            4,566.82           30,313.08
   Jewish congs           1034          20417          1,975      127,480,884.81          123,289.06            6,243.86          123,289.06  
      Episcopal           3034          28726            947      178,483,856.39           42,017.43            6,213.32           42,017.43  
    Hindu congs            182           1014            557        6,582,358.67           36,166.81            6,491.48          700,444.42  


The most interesting column in this table is the amount of `forgiveness provided per job`: there is remarkable similarity between Jewish, Episcopal, and Hindu religious organizations, with an average around $6,300. The amount forgiven to Black churchs is much lower.

The rest of the figures vary widely. Jewish and Episcopal groups seem to employ far more people than do the Black church and Hindu organizations and the numbers of people supported per loan reflect this. Since the PPP was mainly designed to support employemnt, it is not surprising that the amount forgiven is correspondingly higher.

**To return to the initial question: did black churchs recieve less support from the PPP program as some other groups?**

The answer to this is 'yes.' 
- If black churches had received loans in proportion to their representation in the Census, they would have received around 4,600 loans, rather than 552.
- If they received roughly the same amount of money per job, they would have received about $1,750 per job more than they did.

This data doesn't allow us to determine *why* Black churches received fewer loans and for smaller amounts.